staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -565,3 +566,28 @@ func containsString(needle string, haystack []string) bool {
 	}
 	return false
 }
+
+// admissionLogger is a shim around klog that provides additional context based on the admission request.
+// TODO: Replace this with a standard Logger interface when Kubernetes is ready to make the switch.
+type admissionLogger struct {
+	logContext []interface{}
+}
+
+func newAdmissionLogger(attrs Attributes) *admissionLogger {
+	logContext := []interface{}{"operation", attrs.GetOperation(), "resource", attrs.GetResource().String(), "name", attrs.GetName()}
```

Comment:
are there paths that construct this unnecessarily (like ignored subresources, exempt namespaces, etc)?